tests/virt_autotest/login_console.pm

```diff
@@ -209,6 +210,7 @@ sub login_to_console {
     # double-check xen role for xen host
     double_check_xen_role if (is_xen_host and !get_var('REBOOT_AFTER_UPGRADE'));
     check_kvm_modules if is_x86_64 and is_kvm_host and !get_var('REBOOT_AFTER_UPGRADE');
+    update_ssh_console_netconfig if (is_x86_64 and (is_sle or is_sle_micro) and get_var('VIRT_AUTOTEST', ''));
```

Comment:
If this is going to mitigate some of the connection issues to worker machines(eg granache-1) by DNS problem(or DHCP server misbehave by lack of DNS server info), will it help to add it on more test modules? I saw from your mentioned job examples, it does not always happen on the same module, eg some on update_package, while some on host_upgrade_step2_run.

Reply:
host_upgrade_step2_run failed because it already failed in login_console after reboot, the failures can only be seen in some other modules because only those modules try to communicate outside by using FQDN. After fixing name server after reboot, there is no need to do this anymore in other modules.

Reply:
Good finding!